Makefile

```diff
@@ -15,7 +9,7 @@ build1:
 run-bash:
 	docker  run --rm -it --name project-art-run-bash -v ${PWD}:/project/ -v ~/.art/:/root/.art/ project-art-tf2  /bin/bash
 
-run-test:
+
```

Comment:
The `run-test` target removal should be justified. If there is a new way to run tests, it should be documented.
Verify if this change affects any CI/CD pipelines or local development workflows.